src/main/java/gregtech/api/capability/impl/BoilerRecipeLogic.java

```diff
@@ -251,33 +260,33 @@ public MetaTileEntityLargeBoiler getMetaTileEntity() {
     public NBTTagCompound serializeNBT() {
         NBTTagCompound compound = super.serializeNBT();
         compound.setInteger("Heat", currentHeat);
-        compound.setInteger("ExcessFuel", excessFuel);
-        compound.setInteger("ExcessWater", excessWater);
-        compound.setInteger("ExcessProjectedEU", excessProjectedEU);
+        compound.setLong("ExcessFuel", excessFuel);
+        compound.setLong("ExcessWater", excessWater);
+        compound.setLong("ExcessProjectedEU", excessProjectedEU);
         return compound;
     }
 
     @Override
     public void deserializeNBT(@NotNull NBTTagCompound compound) {
         super.deserializeNBT(compound);
         this.currentHeat = compound.getInteger("Heat");
-        this.excessFuel = compound.getInteger("ExcessFuel");
-        this.excessWater = compound.getInteger("ExcessWater");
-        this.excessProjectedEU = compound.getInteger("ExcessProjectedEU");
+        this.excessFuel = compound.getLong("ExcessFuel");
```

Comment:
Do we know how reading a key that was previously stored as an int now as a long behaves? I don't want to void people's boilers because the value failed to read correctly.